control/change_action.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, actions_notifier: ActionNotifier, ui_state: UIState):
         actions_notifier.register(self, act.NewNodeNextSibling)
         actions_notifier.register(self, act.TabNode)
         actions_notifier.register(self, act.UntabNode)
+        actions_notifier.register(self, act.DeleteSelectedNodeAndSelectPrevious)
```

Comment:
Unrelated to the rest of the work - this is something I missed in a previous commit